File: 2270 number-of-ways-to-split-array/solution.java
class Solution {
    public int waysToSplitArray(int[] nums) {
       long sum=0;
       for(int n:nums) sum+=n;
       long pr=0;
       int c=0;
       for(int i=0;i<nums.length-1;i++) {
            pr+=nums[i];
            if(pr>=sum-pr) c++;
       }
       return c;
    }
}

File: 2270 number-of-ways-to-split-array/solution.go
func waysToSplitArray(nums []int) int {
    sum:=0
    for _,v:=range nums {
        sum+=v
    }
    pr,c:=0,0
    for i:=0;i<len(nums)-1;i++ {
        pr+=nums[i]
        if pr>=sum-pr {
            c++
        }
    }
    return c
}
